services/UserService.js

```diff
@@ -25,15 +25,21 @@ module.exports = class UserService {
   }
   //Login
   static async login(body) {
-    console.log("mybody ", body);
+    const usernameOrEmail = body.usernameOrEmail;
+    const password = body.password;
     try {
-      const user = await User.findOne({ where: { email: body.email } });
+      let user;
+      if (usernameOrEmail.includes("@") && usernameOrEmail.includes(".com")) {
```

Comment:
have you test the code, the includes method is not used correctly, includes is for arrays.